frontend/src/Dashboard/HistoricData.js

```diff
@@ -257,96 +342,88 @@ export default function HistoricData() {
                         footerText={dateFormat(weatherData.date)}
                       ></MeasureCard>
                     </div>
-                  </div>
-                );
-              })}
-              ;
-            </MiscGrid>
-          </CardBorder>
-          <CardBorder className="col">
-            <GroupText>Levegő adatok:</GroupText>
-            {airData(weatherData).map((data) => {
-              return (
-                <div className="p-2 m-2" key={data.titleText}>
-                  <MeasureCard
-                    Icon={data.icon}
-                    titleText={data.titleText}
-                    text={data.text}
-                    unit={data.unit}
-                    footerText={dateFormat(weatherData.date)}
-                  ></MeasureCard>
-                </div>
-              );
-            })}
-            ;
-          </CardBorder>
+                  );
+                })}
+              </CardBorder>
+            </div>
+            <CardBorder>
+              <GroupText>Szél adatok:</GroupText>
+              <WindGrid>
+                {windData(weatherData).map((data) => {
+                  return (
+                    <div className="p-2 m-2" key={data.titleText}>
+                      <div className="col">
+                        <MeasureCard
+                          Icon={data.icon}
+                          titleText={data.titleText}
+                          text={data.text}
+                          unit={data.unit}
+                          footerText={dateFormat(weatherData.date)}
+                        ></MeasureCard>
+                      </div>
+                    </div>
+                  );
+                })}
+                ;
+              </WindGrid>
+            </CardBorder>
+            <CardBorder>
+              <GroupText>Talajnedvesség adatok:</GroupText>
+              <SoilGrid>
+                {soilData(weatherData).map((data) => {
+                  return (
+                    <div className="p-2 m-2" key={data.titleText}>
+                      <div className="col">
+                        <MeasureCard
+                          Icon={data.icon}
+                          titleText={data.titleText}
+                          text={data.text}
+                          unit={data.unit}
+                          footerText={dateFormat(weatherData.date)}
+                        ></MeasureCard>
+                      </div>
+                    </div>
+                  );
+                })}
+                ;
+              </SoilGrid>
+            </CardBorder>
+            <CardBorder>
```

Comment:
Hát, ennek örülni fogtok, de a Charthoz hasonlóan ezt a CardBorder-es copy-paste halmazt is ki lehetne szervezni...